Sources/XcodeProj/Scheme/XCScheme+TestableReference.swift

```diff
@@ -17,15 +17,15 @@ public extension XCScheme {
         // MARK: - Init
 
         public init(skipped: Bool,
-                    parallelizable: Bool = false,
+                    parallelization: Parallelization = .none,
                     randomExecutionOrdering: Bool = false,
                     buildableReference: BuildableReference,
                     locationScenarioReference: LocationScenarioReference? = nil,
                     skippedTests: [TestItem] = [],
                     selectedTests: [TestItem] = [],
                     useTestSelectionWhitelist: Bool? = nil) {
```

Comment:
I believe we'd need to create another version of `init` that accepts `parallelizable` as a boolean, so this doesn't cause a breaking change.